client/src/helpers/APICalls/createSchedule.ts

```diff
@@ -0,0 +1,57 @@
+import { AuthApiData } from '../../interface/AuthApiData';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+const createSchedule = async (data: {
+  name: string;
+  days: {
+    monday: {
+      active: boolean;
```

Comment:
Since each day of the week has the exact same properties, we can create a `type` and assign it to avoid repeating ourselves